<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWDialogData</class>
 <widget class="QDialog" name="QWDialogData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑记录</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxRight">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>是否合并</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="dbEditBirth">
        <property name="inputMask">
         <string>9999-99-99 99:99:99</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="dbComboCareer">
        <item>
         <property name="text">
          <string>开发者</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>测试人员</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>产品经理</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>提 交 者</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>职    业</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPlainTextEdit" name="dbEditCodePath"/>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="dbComboIsMerge">
        <item>
         <property name="text">
          <string>是</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>否</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>代码路径</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="dbEditName"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>提交时间</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnOK">
        <property name="text">
         <string>确定</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/images/kiwi_ok.png</normaloff>:/icons/images/kiwi_ok.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>取消</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/images/kiwi_cancel.png</normaloff>:/icons/images/kiwi_cancel.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>329</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>QWDialogData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>QWDialogData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
